data/map systems.txt

```diff
@@ -5648,7 +5649,8 @@ system Arculus
 	fleet "Small Remnant" 2200
 	fleet "Remnant Transport" 2200
 	fleet "Remnant Home Guard" 2000
-	fleet "Korath Ember Waste Raid" 40000
+	fleet "Korath Ember Waste Raid" 120000
+	fleet "Kor Alaji Raid" 60000
```

Comment:
Could you explain what the change is here? I sort of think you may have mentioned it before somewhere, but I'd appreciate if you could explain it again.

Reply:
I think that the change on those two lines is making 2/3 of the korath raids kor alaji raids (I think they're the ones who hunt void sprites).

Reply:
Right, the closer you get to Nenia and the Remnant's home planets, the more you see Alaji.